app/src/test/java/org/simple/clinic/home/patients/PatientScreenRepositoryTest.kt

```diff
@@ -0,0 +1,65 @@
+package org.simple.clinic.home.patients
+
+import com.nhaarman.mockito_kotlin.mock
+import com.nhaarman.mockito_kotlin.whenever
+import io.reactivex.Observable
+import org.junit.Rule
+import org.junit.Test
+import org.simple.clinic.home.patients.illustration.DayOfMonth
+import org.simple.clinic.home.patients.illustration.HomescreenIllustration
+import org.simple.clinic.storage.files.FileStorage
+import org.simple.clinic.storage.files.GetFileResult
+import org.simple.clinic.util.RxErrorsRule
+import org.simple.clinic.util.TestUserClock
+import org.threeten.bp.LocalDate
+import org.threeten.bp.Month
+import java.io.File
+
+class PatientScreenRepositoryTest {
+
+  @get:Rule
+  val rxErrorsRule = RxErrorsRule()
+
+  val userClock = TestUserClock()
+
+  @Test
+  fun `verify correct illustration is picked up`() {
```

Comment:
Nice test! Can we also add a few more test cases, like

- On the bounds of a date range (current date is on the first and last day of the illustration date range)
- When the current date is not part of a illustration date range, it should return a `None`
- When the current date is part of an illustration date, but the downloaded image file is not present, it should return a `None`.